docs/sources/installation/mac.md

```diff
@@ -8,74 +8,43 @@ parent = "installation"
 weight = 500
 +++
 
+# Install on macOS
 
-# Installing on macOS
+This page provides instructions to help you install Grafana on macOS. 
 
-This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
+**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "upgrading.md" >}}) for tips and guidance on updating an existing installation.
 
+## 1. Download and install
 
-## Install using homebrew
+Before you begin, you must have [Homebrew](http://brew.sh/) installed.
 
-Installation can be done using [homebrew](http://brew.sh/)
+1. On the [Grafana download page](https://grafana.com/grafana/download), select the Grafana version you want to install. 
+   * The most recent Grafana version is selected by default.
+   * The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
+2. Select an **Edition**.
+   * **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
+   * **Open Source** - Functionally identical to the enterprise version, but you will need to download the enterprise version if you want enterprise features.
```

Comment:
Would a link here to highlight what the difference between the enterprise features vs. open source are?

Reply:
Currently does not exist. I am trying to get someone to write the page for the website.

Reply:
My understanding is that we don't currently have a homebrew setup for grafana enterprise.

The current enterprise release also does not appear to have osx binaries available, but my understanding is that someone was working on fixing that so we would build osx binaries for future enterprise releases.

For the current stable release there is a tarball published with the oss osx binaries, but the downloads page explicitly does not offer them for download and instead directs folks to use homebrew.  I don't know why that is, but it would be fairly easy to give people the option to download the binaries directly.

Reply:
Nice catch on Enterprise. Updated.